transaction/core/test-utils/src/mint.rs

```diff
@@ -23,16 +23,31 @@ use mc_util_from_random::FromRandom;
 pub fn create_mint_config_tx_and_signers(
     token_id: TokenId,
     rng: &mut (impl RngCore + CryptoRng),
+) -> (MintConfigTx, Vec<Ed25519Pair>) {
+    let mut nonce: Vec<u8> = vec![0u8; NONCE_LENGTH];
+    rng.fill_bytes(&mut nonce);
+
+    create_mint_config_tx_and_signers_for_set_nonce(token_id, rng, nonce)
+}
+
+/// Generate a valid MintConfigTx and return it together with the set of signing
+/// keys that are allowed to sign it.
+///
+/// # Arguments
+/// `token_id` - The token id to use.
+/// `rng` - Randomness source.
+/// `nonce` - Nonce to use
+pub fn create_mint_config_tx_and_signers_for_set_nonce(
+    token_id: TokenId,
+    rng: &mut (impl RngCore + CryptoRng),
+    nonce: Vec<u8>,
```

Comment:
Does this require a separate function that's part of the public API? Feels like it could be an Option parameter on the original function.